src/io/flutter/editor/WidgetIndentGuideDescriptor.java

```diff
@@ -22,25 +27,79 @@
  * edited as there will be a slight delay before new analysis data is available.
  */
 public class WidgetIndentGuideDescriptor {
+  public static class WidgetPropertyDescriptor {
+    private RangeMarker marker;
+    private final FlutterOutlineAttribute attribute;
+
+    WidgetPropertyDescriptor(FlutterOutlineAttribute attribute) {
+      this.attribute = attribute;
+    }
+
+    public String getName() { return attribute.getName();}
+
+    public FlutterOutlineAttribute getAttribute() {
+      return attribute;
+    }
+
+    public int getEndOffset() {
+      if (marker == null) {
+        final Location location = attribute.getValueLocation();
+        final int startOffset = attribute.getValueLocation().getOffset();
```

Comment:
use `location.getOffset()` here since you already have `attribute.getValueLocation()` stored

Reply:
done